certbot-ci/setup.py

```diff
@@ -0,0 +1,50 @@
+from setuptools import setup
+from setuptools import find_packages
+
+
+version = '0.29.0.dev0'
+
+install_requires = [
```

Comment:
we're not using all of these yet

Reply:
No, but next PR will. Do you want me to remove them until that?

Reply:
That would be ideal, to make it easier to track the history of what becomes relevant where.

Reply:
I cleared the dependencies.